demo/app.component.ts

```diff
@@ -39,7 +39,15 @@ declare const alertify: any;
       <div class="tree-container">
         <p class="tree-title">Directory/File structure</p>
         <p class="notice">this tree has advanced configurations</p>
-        <tree
+        <div class="tree-controlls">
+          <p>Try otside manipulations of the tree</p>
+          <button (click)="selectNodeFromDFS(13)">Select 'boot' node</button>
+          <button (click)="switchDFS(2)">Collapse / Expand 'bin' node</button>
```

Comment:
I though it was named after Depth First Search :)) Now I see - it can be as it was. Funny confusion:)